std/encoding.d

```diff
@@ -1466,6 +1462,7 @@ body
     return before - s.length;
 }
 
+///
 unittest
 {
     assert(firstSequence("\u20AC1000") == "\u20AC".length);
```

Comment:
Whoa this unittest sure is scanty. Can we also add a few other examples (of different lengths), even if just for documentation purposes?